src/Views/Textile/Step.elm

```diff
@@ -358,6 +360,50 @@ reparabilityField { current, updateReparability } =
         ]
 
 
+makingComplexityField : Config msg -> Html msg
+makingComplexityField ({ inputs, updateMakingComplexity } as config) =
+    -- Note: This field is only rendered in the detailed step view
+    let
+        makingComplexity =
+            inputs.makingComplexity
+                |> Maybe.withDefault inputs.product.making.complexity
+    in
+    li [ class "list-group-item d-flex align-items-center gap-2" ]
+        [ label [ class "text-nowrap w-25", for "making-complexity" ]
+            [ abbr [ title "Complexité de la confection" ] [ text "Complex" ]
+            , inlineDocumentationLink config Gitbook.TextileMakingComplexity
+            ]
+        , if inputs.knittingProcess == Just Knitting.Seamless then
+            text "Non applicable"
+
+          else
+            [ MakingComplexity.VeryHigh
+            , MakingComplexity.High
+            , MakingComplexity.Medium
+            , MakingComplexity.Low
+            , MakingComplexity.VeryLow
+            ]
+                |> List.map
+                    (\complexity ->
+                        option
+                            [ value <| MakingComplexity.toString complexity
+                            , selected <| complexity == makingComplexity
+                            ]
+                            [ text <| MakingComplexity.toLabel complexity ]
+                    )
+                |> select
+                    [ id "making-complexity"
+                    , class "form-select form-select-sm w-75"
+                    , disabled (inputs.knittingProcess == Just Knitting.FullyFashioned)
+                    , onInput
+                        (MakingComplexity.fromString
+                            >> Result.withDefault inputs.product.making.complexity
+                            >> updateMakingComplexity
+                        )
+                    ]
+        ]
```

Comment:
Seems you reviewed some outdated code, here's how it looks now:

<img width="385" alt="Capture d’écran 2023-05-02 à 11 21 19" src="https://user-images.githubusercontent.com/167767/235628958-84b5120b-adac-409d-8dd6-da1774abef10.png">

Reply:
For some reason, the "Faible" should be better centered vertically. I can take care of that in a next PR if it's too complicated/boring

Reply:
Yeah it seems none of the select boxes are properly vertically aligned, probably since the switch to DSFR styling? I wouldn't mind you fixing it in a future PR (or i'll do it at some point). Thanks

Reply:
Yeah I'll do it 👍